Analysis/src/SVfitAnaInterface.cpp

```diff
@@ -29,8 +29,23 @@ classic_svFit::MeasuredTauLepton CreateMeasuredLepton(const ntuple::TupleLepton&
     }
     if(lepton.leg_type() == analysis::LegType::tau){
         const auto& momentum = lepton.p4();
+        double minVisMass = classic_svFit::electronMass;
+        double maxVisMass = classic_svFit::tauLeptonMass;
```

Comment:
Both minVisMass and maxVisMass are changed below => don't need to initialize them, i.e. `double minVisMass, maxVisMass;`